validator/tests/unit3/test_config/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+__all__ = []
```

Comment:
I'm not entirely sure what the best approach is.  If we actually set it as intended, it seems to mess up API doc generation (maybe setting it to [] does as well).  I removed the line here as it probably isn't necessary for tests.